website_remove_powered_by_odoo/__openerp__.py

```diff
@@ -5,16 +5,17 @@
     'name': 'Website remove Powered by Odoo',
     'version': '8.0.1.0.2',
     'category': 'website',
-    'depends': [
-        'website',
-    ],
+    'depends': ['website'],
     'author': 'Elico Corp',
     'license': 'AGPL-3',
+    'description': """
+        * This module allows you to remove Powered by Odoo from the website.
+        * Note: This module interits the template from the webstie_sale. It may has conflict with certain themes.
```

Comment:
This module inherits the template from the website_sale. It may have conflict with certain themes.